test/parallel/test-tls-inception.js

```diff
@@ -32,8 +32,10 @@ a = tls.createServer(options, function(socket) {
   dest.pipe(socket);
   socket.pipe(dest);
 
-  dest.on('end', function() {
-    socket.destroy();
+  socket.on('error', function(err) {
+    console.error(err);
+    if (err.code !== 'ECONNRESET')
+      throw err;
```

Comment:
Maybe restricting the error handling to windows as in https://github.com/nodejs/node/blob/master/test/parallel/test-child-process-fork-regr-gh-2847.js#L18-L23
Also, probably we can remove the `console.error` as the error info will be available if the exception is thrown